frontend/rollup.config.js

```diff
@@ -43,6 +46,19 @@ export default {
             emitCss: true,
 		}),
 
+		vue({
+            preprocessStyles: true,
+        }),
+        // Replace the environment variables - https://github.com/rollup/rollup/issues/805
+        // btw the vue/dist/vue.esm.browser didn't work
+        replace({
+            'process.env.NODE_ENV': JSON.stringify('development'),
```

Comment:
This should be `"production"` when `production` is true, right? It does seem to save ~30 KiB from the resulting JS archive.

Reply:
Yes, it should by production, my bad, also it can be dynamic based on production value.